codesign/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: codesign
+description: A sample command-line application.
+version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.18.0-170.0.dev <3.0.0'
```

Comment:
I wonder if this will be newer than what is run on CI? In any case, once you update the config file Casey mentioned, we should see if this fails or not.